install

```diff
@@ -30,7 +30,7 @@ fi
 mkdir "$BASEDIR/tanka" && cd "$BASEDIR/tanka"
 
 # Initialise Tanka
-tk init --k8s=1.22
+tk init --k8s=true
```

Comment:
The reason I changed this is that the old command used to throw this error:

```
Error: invalid argument "1.22" for "--k8s" flag: strconv.ParseBool: parsing "1.22": invalid syntax
Usage:
  tk init [flags]

Flags:
  -f, --force   ignore the working directory not being empty
  -h, --help    help for init
      --k8s     set to false to skip installation of k.libsonnet (default true)

Global Flags:
  -v, --verbose

Ouch: invalid argument "1.22" for "--k8s" flag: strconv.ParseBool: parsing "1.22": invalid syntax
```

Reply:
What version of Tanka are you using? It seems like the latest version supports version numbers: https://github.com/grafana/tanka/blob/cc213234f1e103764095e99a3ea974ca71c037cc/cmd/tk/init.go#L57-L65

Reply:
You're right, thank you for pointing this out. I'd installed an old version because the link in the description pointed to an old one. I updated my tanka and also updated the link in the instructions. The command works now as it is!